packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -5,7 +5,6 @@
 
     @include desktop {
         max-height: calc(var(--wallets-vh, 1vh) * 100 - 10rem);
-        overflow: auto;
```

Comment:
FIx issue where dropdown items are breaking and not displaying on top of the modal

Before:

<img width="917" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/6d263515-043c-46c8-84b7-86f73f9eee8e">


After:

<img width="957" alt="Screenshot 2024-05-24 at 12 44 35 PM" src="https://github.com/binary-com/deriv-app/assets/142860499/85ef10c9-f88c-4c60-a90d-b3456458618a">